tools/ratbagd.py

```diff
@@ -252,11 +252,13 @@ class Ratbagd(_RatbagdDBus):
 
     def __init__(self, api_version):
         super().__init__("Manager", None)
-        result = self._get_dbus_property("Devices") or []
-        self._devices = [RatbagdDevice(objpath) for objpath in result]
-        self._proxy.connect("notify::g-name-owner", self._on_name_owner_changed)
+        result = self._get_dbus_property("Devices")
+        if result is None and not self._proxy.get_cached_property_names():
```

Comment:
`result is None` > `not result` unless you have a specific reason.

Reply:
I actually wanted to exclude the case where the list of devices is simply empty. Thus the explicit comparison with `None`. However, in that case the second condition will never trigger. Thus, it actually makes sense to use the simpler comparison.

Reply:
My bad then. I think it's best to leave it as it was if you have the time to change it again. We can wait for @whot though.

Reply:
Undid it for now. :)

Reply:
Thanks!